### PR TITLE
Spanish

### PR DESCRIPTION
**Pull request by tokisuno**

### CODE REVIEW

File: TamoStudy/src/model/language/EnglishLanguage.java [hunk 1]
```diff
@@ -185,7 +185,7 @@ public EnglishLanguage() {
 		sofaText = "Sofa";
 		sunriseText = "Sunrise";
 		nightOutText = "Night Out";
-		enigmaText = "Enigma";
+		enigmaText = "Misterio";
```
Comment (narlock) [inline TamoStudy/src/model/language/EnglishLanguage.java:188]: Remove

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@
 <a href="https://youtube.com/narlock"><img src="https://raw.githubusercontent.com/narlock/TamoStudy/main/README%20Assets/youtubeApp.png" alt="YouTube" align="right" width="60"></a>
 <a href="https://tamostudy.com"><img src="https://raw.githubusercontent.com/narlock/TamoStudy/main/Application%20Assets/icon_512x512%402x.png" alt="TamoStudy logo" title="TamoStudy" align="right" width="60"></a>
 
-# TamoStudy
+# TamoStudy --- Spanish Translation
```
Comment (narlock) [inline README.md:9]: If you are interested in creating a spanish translation of the readme, can you make a README_ES.md file and translate the readme?

This line needs to be remove from this

**Review by narlock — CHANGES_REQUESTED** [review]

File: TamoStudy/src/model/language/EnglishLanguage.java (hunk 1 above)
Comment (tokisuno) [reply TamoStudy/src/model/language/EnglishLanguage.java:188]: remove..? Mistero? you're removing the engimaText entirely?

File: README.md (hunk 2 above)
Comment (tokisuno) [reply README.md:9]: yeah ngl i forgot i wrote that in. wrote that in when i was figuring out wtf i was doing LMAO.

File: TamoStudy/src/model/language/EnglishLanguage.java (hunk 1 above)
Comment (narlock) [reply TamoStudy/src/model/language/EnglishLanguage.java:188]: No, the file that you edited was `EnglishLanguage.java` in this change

**Review by narlock — APPROVED** [review]

**Pull request merged**